src/lib.ts

```diff
@@ -429,6 +433,10 @@ export function printHelpMenu(): void {
     +--------------+---------------------------------------------------------------------------------------------------+
     | &lt;somenumber&gt; | garbo will terminate after the specified number of turns, e.g. \`garbo 200\` will terminate after   |
     |              |  200 turns are spent. Negative inputs will cause garbo to terminate when the specified number of turns remain.       |
+    +------------------------------------------------------------------------------------------------------------------+
+    |   simdiet    | garbo will print out what it computes as an optimal diet and then exit                            |
+    +------------------------------------------------------------------------------------------------------------------+
+    |    nodiet    | *EXPERIMENTAL* garbo will not eat or drink anything as a part of its run (including pantsgiving)  |
```

Comment:
add these to README, if you dare

Reply:
We can wait to push before doing this; you can declare this out of scope of this PR.